src/frontends/onnx/tests/tests_python/test_backend.py

```diff
@@ -482,17 +482,12 @@ def expect_fail(test_case_path, xfail):  # type: (str) -> None
         "OnnxBackendNodeModelTest.test_reduce_l2_keep_dims_random_cpu",
         "OnnxBackendNodeModelTest.test_reduce_l2_negative_axes_keep_dims_example_cpu",
         "OnnxBackendNodeModelTest.test_reduce_l2_negative_axes_keep_dims_random_cpu",
-        "OnnxBackendNodeModelTest.test_reduce_log_sum_asc_axes_cpu",
```

Comment:
Regarding issues with tests. I took a look on a models. They fails for your particular case because axes has more than 1 item.

You can take a look on a source scripts: https://github.com/onnx/onnx/blob/0bb2775098cb72c5619451f37cc0c8f9f335893f/onnx/backend/test/case/node/reduce_log_sum.py#L14 it has two axes for reduction.

Currently, such tests are failing (known issue). It isn't issue in your code, but it is blocking merge.

We should return back these four tests:
```
        "OnnxBackendNodeModelTest.test_reduce_log_sum_asc_axes_cpu",
        "OnnxBackendNodeModelTest.test_reduce_log_sum_asc_axes_expanded_cpu",
        "OnnxBackendNodeModelTest.test_reduce_log_sum_desc_axes_cpu",
        "OnnxBackendNodeModelTest.test_reduce_log_sum_desc_axes_expanded_cpu",
```